android/src/main/java/io/radar/capacitor/RadarPlugin.java

```diff
@@ -150,27 +152,83 @@ public void initialize(PluginCall call) {
         call.resolve();
     }
 
+    @PluginMethod()
+    public void setLogLevel(PluginCall call) {
+        String level = call.getString("level");
+        Radar.RadarLogLevel logLevel = Radar.RadarLogLevel.NONE;
+        if (level != null) {
+            level = level.toLowerCase();
+            if (level.equals("error")) {
+                logLevel = Radar.RadarLogLevel.ERROR;
+            } else if (level.equals("warning")) {
+                logLevel = Radar.RadarLogLevel.WARNING;
+            } else if (level.equals("info")) {
+                logLevel = Radar.RadarLogLevel.INFO;
+            } else if (level.equals("debug")) {
+                logLevel = Radar.RadarLogLevel.DEBUG;
+            } else {
+                call.reject("bad request");
+                return;
+            }
+        }
+        Radar.setLogLevel(logLevel);
+        call.resolve();
+    }
+
     @PluginMethod()
     public void setUserId(PluginCall call) {
         String userId = call.getString("userId");
         Radar.setUserId(userId);
         call.resolve();
     }
 
+    @PluginMethod()
+    public void getUserId(PluginCall call) {
+        JSObject ret = new JSObject();
+        ret.put("userId", Radar.getUserId());
+        call.resolve(ret);
+    }
+
     @PluginMethod()
     public void setDescription(PluginCall call) {
         String description = call.getString("description");
         Radar.setDescription(description);
         call.resolve();
     }
 
+    @PluginMethod()
+    public void getDescription(PluginCall call) {
+        JSObject ret = new JSObject();
+        ret.put("description", Radar.getDescription());
+        call.resolve(ret);
+    }
+
     @PluginMethod()
     public void setMetadata(PluginCall call) {
         JSObject metadata = call.getObject("metadata");
         Radar.setMetadata(RadarPlugin.jsonObjectForJSObject(metadata));
         call.resolve();
     }
 
+    @PluginMethod()
+    public void getMetadata(PluginCall call) {
+        call.resolve(RadarPlugin.jsObjectForJSONObject(Radar.getMetadata()));
+    }
+
+    @PluginMethod()
+    public void setAnonymousTrackingEnabled(PluginCall call) {
+        boolean enabled = call.getBoolean("enabled");
```

Comment:
Let's `reject` is `enabled` is missing to keep it consistent with iOS

Reply:
@tjulien One thing important is that capacitor module is written by Typescript, so `enabled` can't be missing.